goreadme.go

```diff
@@ -207,6 +209,19 @@ func (r *GoReadme) get(ctx context.Context, name string) (*pkg, error) {
 			}
 		}
 	}
+	if !r.config.Types {
```

Comment:
Can you add a similar comment to the one above?

```go
	// If types were not requested to be added to the readme, add their
	// examples to the main readme.
```

Reply:
Yes